sonar-php-plugin/src/main/java/org/sonar/plugins/php/api/Php.java

```diff
@@ -36,12 +36,12 @@
 
   public static final String DEFAULT_FILE_SUFFIXES = "php,php3,php4,php5,phtml,inc";
 
-  private Configuration settings;
+  private Settings settings;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "Settings"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)